bots/employee-finder/src/SSW.SophieBot/language-understanding/en-us/SSWSophieBot.en-us.lu

```diff
@@ -326,6 +326,7 @@
 - newcastle list
 - show people based on sydney
 - who is remote
+- who works in the Sydney office?
```

Comment:
Resolves https://github.com/SSWConsulting/SSW.SophieBot/issues/657#issue-1978659561

Reply:
Consider using pattern or intent feature rather than statically coding the phrase for each site.
Pattern example: who work[s] {@prep} [the] ({@geographyV2}|{@site}) [office][?]^
Feature example: @ intent GetPeopleBasedOnLocation usesFeature geographyV2,site

Reply:
Done - Used pattern and tested pass locally. See: https://github.com/SSWConsulting/SSW.SophieBot/pull/658/commits/d3da12e3ad6d323358a33855913f2d4ef4c33186

![image](https://github.com/SSWConsulting/SSW.SophieBot/assets/37203901/847700ff-3ac3-408f-8043-5a2b9058d120)